fctesting/utils.go

```diff
@@ -43,3 +46,24 @@ func RequiresRoot(t testing.TB) {
 			rootDisableEnvName)
 	}
 }
+
+func newLogger() *log.Logger {
+	str := os.Getenv(logLevelEnvName)
+	if str != "" {
+		logLevel, err := log.ParseLevel(str)
+		if err != nil {
+			panic(err)
+		}
+		return &log.Logger{
+			Out:   os.Stdout,
+			Level: logLevel,
+		}
+	} else {
+		return log.New()
+	}
+}
+
+// NewLogEntry creates log.Entry. The level is specified by "FC_TEST_LOG_LEVEL" environment variable
+func NewLogEntry() *log.Entry {
```

Comment:
This should probably take a `testing.TB` and pass that to `newLogger`. This will get rid of the panic and instead call `t.Fatalf`